frontend/packages/console-shared/src/utils/__mocks__/resource-utils-test-data.ts

```diff
@@ -0,0 +1,119 @@
+import { Alerts } from '@console/internal/components/monitoring/types';
+import { AlertStates, RuleStates } from '@console/internal/reducers/monitoring';
+
+export const mockAlerts: Alerts = {
```

Comment:
We can move this [rules-mocks.ts](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/monitoring/alerts/__tests__/rules-mocks.ts) to console-shared and use it at both the places. WDYT?

Reply:
created `alerts-and-rules-data.ts` under shared utils